src/fides/api/service/saas_request/saas_request_override_factory.py

```diff
@@ -24,6 +25,14 @@ class SaaSRequestType(Enum):
     DELETE = "delete"
     OPT_IN = "opt_in"
     OPT_OUT = "opt_out"
+    GET_CONSENTABLE_ITEMS = "get_consentable_items"
+    UPDATE_CONSENT = "update_consent"
+    PROCESS_CONSENT_WEBHOOK = "process_consent_webhook"
```

Comment:
One request type for each of the new bidirectional consent workflows
- get consentable items
- update consent preferences
- process consent webhook